### ISSUE TITLE
How to NOT render the title global attribute.

### ISSUE THREAD

**Issue by shyakadavis** [metonym/svelte-time#43]
Hello, 👋 

First off, I **really** love the lib and use it often for its simplicity. 🙂 

From MDN, I understand the `title` attribute is basically a tooltip for representing advisory information related to the element it belongs to. In our case, the `time` element. (Thanks for making this semantic.)

But here is my issue; I wanted to provide my own tooltip, but trying to _override_ the rendered title doesn't seem to work.

I tried passing `null` and an empty string `''` to the `title` prop, but nothing seemed to work.

```svelte
						<Time
							relative
							format="D MMMM, YYYY"
							timestamp={created_at}
							class="text-sm text-muted-foreground"
							title={null}
						/>
```

The result, in my case, is repetitive information and I would like to avoid this:

<img width="271" alt="Screenshot 2024-04-18 at 19 54 25" src="https://github.com/metonym/svelte-time/assets/87414827/ac5958eb-270d-4bc5-82f6-892c890f9012">

---

Do you know of a way to:

1. disable the tooltip, but retain the element title in the DOM (for accessibility's sake?)
2) forego on adding the title altogether

I appreciate the help.

**Comment by metonym** [comment]
Hey – thank you for the feedback and taking the time to open this issue. I'm working on a PR that will allow the `title` attribute to be overridden.

**Comment by metonym** [comment]
Released in [v0.9.0](https://github.com/metonym/svelte-time/releases/tag/v0.9.0).

You can now specify a custom `title` or omit it altogether by setting `title={undefined}`.

```svelte
<Time relative title="Custom title" />
```

Set the value to `undefined` to omit the `title` altogether.

```svelte
<Time relative title={undefined} />
```

**Comment by shyakadavis** [comment]
Hello, @metonym 

Thank you very much for the release. It works splendidly.

I appreciate the help.

Be safe. 🙂
